src/test/java/com/marklogic/spark/reader/optic/InferSchemaTest.java

```diff
@@ -36,7 +36,7 @@ void inferFromAllTdeTypes() throws Exception {
         String columnInfoResponse = readClasspathFile("allTypes-columnInfo-response.txt");
         StructType schema = SchemaInferrer.inferSchema(columnInfoResponse);
 
-        assertEquals(35, schema.size(), "The TDE has 35 columns, and the hidden 'rowid' column that's returned " +
+        assertEquals(36, schema.size(), "The TDE has 35 columns, and the hidden 'rowid' column that's returned " +
```

Comment:
Pretty small nit, but does the message also need to be updated?

Reply:
Not a nit, the message is wrong. Updating.